small-talk/app/components/friends/individualFriend.js

```diff
@@ -4,6 +4,7 @@ function IndividualFriend({ details }) {
     return (
         <div className="border p-4 bg-gray-100 rounded-lg flex items-center justify-between">
             {/* add profileIcon component here */}
+            <img src={details.img} width="50" height="50"></img>
```

Comment:
Was wondering if you could import the ProfileIcon component here and use `<ProfileIcon avatarID={details.avatar} backgroundID={details.background} avatarClass={`w-50`} backgroundClass={p-#}/>` so there is a background included with the display.